dp/cloud/go/active_mode_controller/internal/message_generator/sas/registration.go

```diff
@@ -9,16 +13,57 @@ func NewRegistrationRequestGenerator() *registrationRequestGenerator {
 }
 
 func (*registrationRequestGenerator) GenerateRequests(cbsd *active_mode.Cbsd) []*Request {
-	req := &registrationRequest{
-		UserId:           cbsd.GetUserId(),
-		FccId:            cbsd.GetFccId(),
-		CbsdSerialNumber: cbsd.GetSerialNumber(),
-	}
+	req := buildRegistrationRequest(cbsd)
```

Comment:
i think we shouldn't be generating a registration request for a radio unless it's either marked as "multi-step" or it's single-step && type A && has installation params filled out && is indoors (for now). These fields, in a single-step should be updated by enodebd first before we send a request.